vcbuild.bat

```diff
@@ -4,14 +4,9 @@
 :: explicitly allow them to persist in the calling shell.
 endlocal
 
-if /i "%1"=="help" goto help
-if /i "%1"=="--help" goto help
-if /i "%1"=="-help" goto help
-if /i "%1"=="/help" goto help
-if /i "%1"=="?" goto help
-if /i "%1"=="-?" goto help
-if /i "%1"=="--?" goto help
-if /i "%1"=="/?" goto help
+set "arg=%1"
+if /i "%arg:~-1%"=="?" goto help
+if /i "%arg%"=="help" goto help
```

Comment:
I'm just curious why wasn't this part modified similar to the `?` modification eg. `if /i "%arg:~-4%"=="help" goto help`? With the way it is now `--help`, `-help`, and `/help` will not get caught.

Reply:
Haha how did I miss that 🤔